KOMODO-IMPRESS/Assets/IMPRESS/Scripts/WorldPulling.cs

```diff
@@ -256,29 +376,36 @@ public Vector3 ComputePositionDifference (UpdatingValue<Vector3> handsAveragePos
             return handsAveragePosition.Initial - handsAveragePosition.Current;
         }
 
-        public float ComputeScale (UpdatingValue<float> handDistance, UpdatingValue<float> playerLocalScaleX)
+        // Computes the dif
```

Comment:
Finish the comment

Reply:
I found out the function is not used, so I deleted the function